modules/virt-about-hostpath-provisioner.adoc

```diff
@@ -6,22 +6,8 @@
 [id="virt-about-hostpath-provisioner_{context}"]
 = About the hostpath provisioner
 
-The hostpath provisioner is a local storage provisioner designed for
-{VirtProductName}. If you want to configure local storage for
-virtual machines, you must enable the hostpath provisioner first.
+When you install the {VirtProductName} Operator, the hostpath provisioner (HPP) Operator is automatically installed. The HPP is a local storage provisioner designed for {VirtProductName}. If you want to configure local storage for virtual machines, you must enable the HPP first.
 
-When you install the {VirtProductName} Operator, the hostpath provisioner Operator
-is automatically installed. To use it, you must:
+In {VirtProductName} 4.10, the HPP is implemented using a Kubernetes CSI driver. The CSI driver enables you to configure local storage for your virtual machines with much less performance overhead, such as a restart of all {product-title} nodes because the `MachineConfig` object is no longer used to write to the directory on the host.
```

Comment:
I think you can leave it at just saying that in 4.10 it is implemented using a Kubernetes CSI driver. I don't think we know if there is less performance overhead or not. Mentioning MachineConfig is not really useful here, it makes sense to say in the release notes it is not longer needed but it is not relevant for the user documentation.

Reply:
Thanks.